### DISCUSSION TITLE
frpc源码是否开放（或者是否 有RISC-V版本）？

### DISCUSSION THREAD

**Discussion by bitmoeinc** [natfrp/wiki#237] [水深火热]
frpc源码是否开放（或者是否 有RISC-V版本）？
搞来了个新开发版，RISC-V架构的，没找到可用的客户端，谢谢

**Comment by fakeboboliu** [comment]
我还没收到在 kickstarter 买的 VisionFive 2，所以在 release 中还没有包含。

如果您有兴趣的话，[docker 镜像](https://hub.docker.com/layers/natfrp/frpc/latest/images/sha256-e09c9fbe3c1df7c852532008e2c660bb3819cd0baeaf3b0701d87c46dc33ec71?context=repo) 现已包含 riscv64 版本，但是我们很难确定它在实机上的运行情况。
